site/docs/v1/tech/apis/_webhook-request-body.adoc

```diff
@@ -46,6 +48,9 @@ The read timeout in milliseconds used when FusionAuth sends events to the Webhoo
 [field]#webhook.sslCertificate# [type]#[String]# [optional]#Optional#::
 An SSL certificate in PEM format that is used to establish the SSL (TLS specifically) connection to the Webhook.
 
+[field]#webhook.tenantIds# [type]#[Array<UUID>]# [optional]#Optional# [since]#Available since 1.37.0#::
```

Comment:
Should webhook request/response descriptions for `tenantIds` be updated to reference the `applicationIds` field to help people on older versions find the correct field?